Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -107,6 +120,11 @@ class TextureCacheBase
     u32 GetNumLevels() const { return texture->GetConfig().levels; }
     u32 GetNumLayers() const { return texture->GetConfig().layers; }
     AbstractTextureFormat GetFormat() const { return texture->GetConfig().format; }
+    u32 InMemoryFormat() const { return format & 0xff; }
```

Comment:
Phire basically summed it up but yes that in memory format for the hybrid texture is still there in the hybrid PR.  And it seems like there are other formats that are 6bits as well.  Just look the enumerations for `TextureFormat`